server/integration/testsuite/pom.xml

```diff
@@ -359,7 +358,6 @@
         </groups.client.local.domain>
         <groups.client.clustered>
             org.infinispan.server.test.category.HotRodClustered,
-            org.infinispan.server.test.category.MemcachedClustered,
```

Comment:
You're removing it here but not removing it from suite.client.dist.domain.phase and suite.client.local.domain.phase so the  domain profile will be broken, IMO.